### PR TITLE
Bibtex referencing

### LINKED ISSUE #519: Edit references consistently
### Summary

References are currently inconsistent - going forward it might be nice to agree to one style.

### What needs to be done?

Pick a referencing style - please vote below
* :rocket: for APA e.g. Several studies (Miller, 1999; Shafranske & Mahoney, 1998) and Buckhoff, T. A., & Colson, R. H. (2003). Preventing fraud by conducting background checks. CPA Journal, 73(11), 52
* :+1: for Chicago "After that I lived like a young rajah in all the capitals of Europe…" (Fitzgerald, 2004). and Fitzgerald, F. (2004). The great Gatsby. New York: Scribner. 

Any others that people like?

### Who can help?

Everyone to stop this being a librarian fight!

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by masonlr**
### Summary

Fixes #519 

### List of changes proposed in this PR (pull-request)

* Adds `book/website/_bibliography/references.bib`
* Adds instructions to contributors concerning referencing

### What should a reviewer concentrate their feedback on?

- [x] References show correctly on netlify build
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by masonlr** [comment]
@cassgvp I've added a section here for contributors https://github.com/alan-turing-institute/the-turing-way/blob/6e88ac01391ab932e18c6a0f953e7f65e16cb6e6/CONTRIBUTING.md#referencing-and-citing

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,29 +2,43 @@
 
 :tada::balloon::cake: **Welcome to the Turing Way repository!** :cake::balloon::tada:
 
-:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+:dizzy::hatched*chick::sunny: \_We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
```
Comment (rainsworth) [inline CONTRIBUTING.md]:
```suggestion
:dizzy::hatched_chick::sunny: _We're so excited you're here and want to contribute._ :sunny::hatched_chick::dizzy:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,29 +2,43 @@
...
+  - [Joining the community](#joining-the-community)
+  - [Inclusivity](#inclusivity)
+  - [Get in touch](#get-in-touch)
+  - [Contributing through GitHub](#contributing-through-github)
+  - [Writing in Markdown](#writing-in-markdown)
+  - [Where to start: issues](#where-to-start-issues)
+    - [Issue labels](#issue-labels)
+  - [Making a change with a pull request](#making-a-change-with-a-pull-request)
+    - [1. Comment on an existing issue or open a new issue referencing your addition](#1-comment-on-an-existing-issue-or-open-a-new-issue-referencing-your-addition)
+    - [2. [Fork][github-fork] the [Turing Way repository][turing-way-repo] to your profile](#2-forkgithub-fork-the-turing-way-repositoryturing-way-repo-to-your-profile)
```
Comment (rainsworth) [inline CONTRIBUTING.md]: ToC link broken - perhaps removing links within header will fix it?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,29 +2,43 @@
...
+    - [3. Make the changes you've discussed](#3-make-the-changes-youve-discussed)
+    - [4. Submit a [pull request][github-pullrequest]](#4-submit-a-pull-requestgithub-pullrequest)
```
Comment (rainsworth) [inline CONTRIBUTING.md]: ToC link broken - perhaps removing links within header will fix it?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -73,7 +88,7 @@ GitHub has a helpful page on [getting started with writing and formatting on Git
 
 Most of the writing that you'll do will be in [Markdown][markdown].
 You can think of Markdown as a few little symbols around your text that will allow GitHub to render the text with a little bit of formatting.
-For example you could write words as **bold** (`**bold**`), or in *italics* (`*italics*`), or as a [link][rick-roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
+For example you could write words as **bold** (`**bold**`), or in _italics_ (`*italics*`), or as a [link][rick-roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
```
Comment (rainsworth) [inline CONTRIBUTING.md]:
```suggestion
For example you could write words as **bold** (`**bold**`), or in _italics_ (`_italics_`), or as a [link][rick-roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (masonlr) [reply CONTRIBUTING.md]: Ha, this looks like nested links... Inception style!

Comment (masonlr) [reply CONTRIBUTING.md]: This is an issue with the "Markdown All in One" extension on VS Code. The Prettier extension works fine.

File: CONTRIBUTING.md (hunk 1 above)
Comment (masonlr) [reply CONTRIBUTING.md]: Yep. Looks good.

**Comment by masonlr** [comment]
@rainsworth the emoji issue was with prettier, if this keeps coming up the solution is:

```markdown
<!-- prettier-ignore -->
:dizzy::hatched_chick::sunny: _We're so excited you're here and want to contribute._ :sunny::hatched_chick::dizzy:
```

**Comment by rainsworth** [comment]
@masonlr thanks! But now the [table of contents indents have gone funny](https://github.com/alan-turing-institute/the-turing-way/blob/670106217ca87caba9a48a3ad0eee8d8a1be8432/CONTRIBUTING.md)?

**Comment by masonlr** [comment]
> @masonlr thanks! But now the [table of contents indents have gone funny](https://github.com/alan-turing-institute/the-turing-way/blob/670106217ca87caba9a48a3ad0eee8d8a1be8432/CONTRIBUTING.md)?

@rainsworth How's it [now](https://github.com/alan-turing-institute/the-turing-way/blob/bibtex-referencing/CONTRIBUTING.md)? 

This again was the rogue vs code extension making its own TOC.

**Comment by rainsworth** [comment]
> @rainsworth How's it [now](https://github.com/alan-turing-institute/the-turing-way/blob/bibtex-referencing/CONTRIBUTING.md)?
> 
> This again was the rogue vs code extension making its own TOC.

The indents are fixed! I'm now just being picky about the spaces in between lines 🙈 Is there a way to not have extra paragraphs within the "Making a change with a pull request" section (and I think after the initial "Contributing to the Turing Way"? Sorry if I'm being a pain - I can fix these manually but is this due to something that was automated? (Still trying to catch up on everything added during the book dash!)

**Comment by masonlr** [comment]
Can you post a screenshot of the section?

**Comment by masonlr** [comment]
Do you mean? 

image

image

**Comment by rainsworth** [comment]
Screenshot 2019-05-30 at 14 49 20

vs

Screenshot 2019-05-30 at 14 51 15

**Comment by masonlr** [comment]
Yeah for sure. That huge detailed table of contents was generated from a rogue markdown extension that I've since turned off.

**Comment by rainsworth** [comment]
> Yeah for sure. That huge detailed table of contents was generated from a rogue markdown extension that I've since turned off.

Sorry, I didn't mean replace with original ToC, I just meant the added white space in between items in the list - I thought it arose because there were extra paragraphs after some of the items which then spaced out the entire ToC. I just didn't know if they were added from the automated reformatting or what was causing it!

Sorry if I'm not making sense, I just can't see from this PR where the changes to the automation are being made (if that's what's happening) - so I'm just a bit confused!

**Comment by masonlr** [comment]
Okay. I think the spacing might just be how github renders the markdown when there are multi-level lists. I'll make a throw-away test and screenshot it.

**Comment by masonlr** [comment]
Ah, think this finally gets there:

image

**Comment by rainsworth** [comment]
Yes! I am so sorry to have been such a pain about that 🙈 , I know it's something I could fix manually I just wasn't sure how it would affect the rest of the repo!

**Comment by masonlr** [comment]
Yep. The confusion is (i) editor extensions which do rogue formatting and (ii) the prettier formatter. Going forward, prettier.io is solid and should solve a bunch of these random formatting issue that may crop up.

**Comment by masonlr** [comment]
There's no automation in there as yet.

**Comment by rainsworth** [comment]
Thanks so much @masonlr for all this. It looks good except there seem to be some merge conflicts - are you ok to resolve?

I'm also wondering if it would be better for us to review & merge https://github.com/alan-turing-institute/the-turing-way/pull/568 first? As that will definitely conflict.

**Comment by masonlr** [comment]
@rainsworth, go ahead and merge #568. I'll wait for that, then rebase + fix all the conflicts.

**Comment by rainsworth** [comment]
> @rainsworth, go ahead and merge #568. I'll wait for that, then rebase + fix all the conflicts.

OK, that's merged in. Let me know when you're ready for review! Thank you again, sorry for all the trouble - really appreciate your contributions 👏

**Comment by masonlr** [comment]
@rainsworth, the present PR should now contain all of @SusannaSansone's changes (from #568) with a layer of bibtex referencing added in.

**Pull request merged**